test/ignored-changes.json

```diff
@@ -0,0 +1,57 @@
+[
```

Comment:
Nit: all other revapi configuration files are called `revapi.json`.

Reply:
I kept this as we previously had `ignored-changes.xml` where we would put the temporarily ignored changes. This file is cleaned out on every release. You can read more about it here: https://github.com/camunda-cloud/zeebe/wiki/Breaking-Changes#java (I will of course update this when we switch to Revapi :sweat_smile:)